purescriptbook.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "path": "."
+    },
+    {
+      "path": "exercises/chapter2"
+    },
+    {
+      "path": "exercises/chapter3"
+    },
+    {
+      "path": "exercises/chapter4"
+    },
+    {
+      "path": "exercises/chapter5"
+    },
+    {
+      "path": "exercises/chapter6"
+    },
+    {
+      "path": "exercises/chapter7"
+    },
+    {
+      "path": "exercises/chapter8"
+    },
+    {
+      "path": "exercises/chapter9"
+    },
+    {
+      "path": "exercises/chapter10"
+    },
+    {
+      "path": "exercises/chapter11"
+    },
+    {
+      "path": "exercises/chapter12"
+    },
+    {
+      "path": "exercises/chapter13"
+    },
+    {
+      "path": "exercises/chapter14"
+    },
+    {
+      "path": "tmp/purcel"
+    },
+    {
+      "path": "tmp/purescript-comonad-ui-todos"
+    }
+  ],
+  "settings": {
+    "files.exclude": {
+      "**/.git": true,
+      "**/.svn": true,
+      "**/.hg": true,
+      "**/CVS": true,
+      "**/.DS_Store": true,
+      "exercises": true
+    }
+  },
+  "extensions": {
+    "recommendations": [
+      "nwolverson.ide-purescript",
+      "nwolverson.language-purescript",
+      "mvakula.vscode-purty"
+    ]
+  }
```

Comment:
I only have the first two extensions installed. Should the purty extension be recommended if I open a project with PR applied?

Reply:
When you open the folder you should get a prompt to open the workspace.
If you say yes or open directly you will get a prompt to install the extensions.
And opening a purescript file will give you a prompt to build the subproject. When built language support works. With some bugs supporting multiple folders at the same time but better than nothing

Reply:
I cleaned up the workspace.
After opening a file with the workspace:
![image](https://user-images.githubusercontent.com/2453277/128616491-5dcb48f1-e6ef-4f95-86dc-37908bfb4581.png)
...and after building:
![image](https://user-images.githubusercontent.com/2453277/128616512-8f7fb70c-6b9c-4067-8ab6-0bc56e174b0b.png)